### ISSUE TITLE
WinMain loop does not sleep, making fixedFrameRate unplayably fast

### ISSUE THREAD

**Issue by trigger-segfault** [trigger-segfault/OpenLRR#10]
### Describe the issue

Similar to other engine issues (such as https://github.com/trigger-segfault/OpenLRR/issues/9), the concept of elapsed time is merely a *delta* of the last update, but with no real restrictions on the *number* of updates over a period of time.

The command-line option: `-fpslock <float>` exists to enforce a fixed frame rate by setting the `engine/Main` global variable `fixedFrameRate`. This variable is used during the main loop within `engine/Main` **&gt;** `WinMain` function, where the time delta is always set to the value of `fixedFrameRate`, and used in the next MainLoop update call.

*But the WinMain loop does not sleep* (or wait at all). So although this has the correct effect on game logic of using a consistent time delta for every update... the **real-time** speed of the game (and *how often* it updates per second) is entirely at the mercy of your computer's performance.

Simply put: This does not change the **number** of updates per second, this simply **reports** what the number of updates per second **should be** to the MainLoop update call.

### Steps to fix

Determine a good method for the WinMain loop to wait until the next frame (when fixed update rate is being used). This waiting also needs to take place for other main states affecting the frame rate (i.e. `mainGlobs.flags & MAIN_FLAG_DUMPMODE`).

**Comment by trigger-segfault** [comment]
This has been implemented by 09b54266e1c70831493821782e7a55e38adab449.
